index.html

```diff
@@ -31,7 +32,7 @@
       property="twitter:description"
       content=" Web3Collectives is a free library of over 200 web3 resources on varying topics like Smart contract deployment, building decentralized applications, cryptography fundementals and lots more..."
     />
-    <meta property="twitter:image" content="https://" />
+    <meta property="twitter:image:src" content="/Web3Collectives.png" />
 
     <!-- Favicons -->
     <link rel="icon" sizes="16x16" href="./src//assets//images/logo-16.png" />
```

Comment:
It looks like the href path is not right 
Should be something like `./src/assets/images/logo-16.png`

Reply:
Please confirm this.

Reply:
Ok